models/games.get.models.js

```diff
@@ -33,3 +33,31 @@ exports.allUsers = () => {
     })
     
 }
+
+exports.allReviews = (category) => {
+    
+
+        
+    let queryStr = `SELECT reviews.review_id, reviews.title, reviews.designer, reviews.owner, reviews.review_img_url, reviews.review_body, reviews.category, reviews.created_at, reviews.votes, COUNT(comments.review_id)::INT AS comment_count FROM reviews JOIN comments ON reviews.review_id = comments.review_id `;
```

Comment:
Double check the Trello instructions... it doesn't want you to bring the review_body back in the query